specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachineScaleSet.json

```diff
@@ -3925,6 +3931,13 @@
             "lun"
           ],
           "description": "Specifies the parameters that are used to add data disks to the virtual machines in the scale set. <br><br> For more information about disks, see [About disks and VHDs for Azure virtual machines](https://docs.microsoft.com/azure/virtual-machines/managed-disks-overview)."
+        },
+        "diskControllerType": {
+          "type": "string",
+          "items": {
+            "$ref": "./computeRPCommon.json#/definitions/DiskControllerType",
+            "description": "Specifies the disk controller type configured for the virtual machines in the scale set. This property is only supported for virtual machines whose operating system disk and and VM size supports hyperVGeneration: V2. For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes.<br><br> Minimum api-version: 2020-08-01"
```

Comment:
Minimum api-version need to be 2022-08-01

Reply:
Also remove the below as it is covered in the DiskControllerType definition.

This property is only supported for virtual machines whose operating system disk and and VM size supports hyperVGeneration: V2. For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes.

Reply:
acknowledging